project-base/storefront/components/Blocks/Login/LoginForm.tsx

```diff
@@ -115,38 +147,9 @@ export const LoginForm: FC<LoginFormProps> = ({
                             </FormButtonWrapper>
 
                             <div className="flex gap-2 lg:order-2 w-full justify-center">
```

Comment:
I would consider checking the length of the `settingsData?.settings?.socialNetworkLoginConfig` to prevent an empty div if there is no data.

Reply:
:ok_hand: fixed in 98beaf69d1